README.md

```diff
@@ -15,9 +15,8 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 1.	Make sure you have `git`, `node`, and `npm` installed.
 2.	Clone this repository locally.
 3.	Add `127.0.0.1 calypso.localhost` to your local `hosts` file.
-4.	Run `npm install`
-5.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
-6.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.
+4.	Execute `npm start` or `npm run dashboard` (for a more visually-oriented interface) from the root directory of the repository.
```

Comment:
On a fresh start (without any package installed) `npm run dashboard` doesn't work because `webpack-dashboard` is not installed. Shall we update the `dashboard` command to `"dashboard": "npm install && webpack-dashboard -- npm start",`? This is referenced both in this `README` and the `docs/install.md`